.github/workflows/TestRunnerTiledSiracusa.yml

```diff
@@ -47,11 +47,16 @@ jobs:
           path: /app/.ccache
           key: ${{ runner.os }}-ccache
       - name: Run Test
-        run: |
-          cd DeeployTest
-          mkdir -p /app/.ccache
-          export CCACHE_DIR=/app/.ccache
-          source /app/install/pulp-sdk/configs/siracusa.sh
-          python testRunner_tiled_siracusa.py -t Tests/${{ inputs.test-name }} --cores=${{ inputs.num-cores }} --l1 ${{ matrix.L1 }} --defaultMemLevel=${{ inputs.default-memory-level }} ${{ inputs.double-buffer && '--doublebuffer' || '' }}
-        shell: bash
+        uses: nick-fields/retry@v3
```

Comment:
Do we have more insight into why this test times out sometimes?

Reply:
But at which point is it happening? When running Deeploy or during the simulation with GVSoC?

Reply:
It happens during compilation :/

Reply:
Okay, then I suggest you open an issue, reference this conversation/changes as a temporary bug fix in the issue, and merge this PR.

Reply:
Sounds resonable